erp/jobs/import_centres_vaccination.py

```diff
@@ -0,0 +1,117 @@
+"Imports vaccination centers from open data: https://www.data.gouv.fr/fr/datasets/lieux-de-vaccination-contre-la-covid-19/"
+
+from django.conf import settings
+import requests
+import logging
+
+from core import mailer
+from erp.mapper.vaccination import RecordMapper
+from erp.models import Activite
+
+logger = logging.getLogger(__name__)
+
+API_URL = (
+    "https://www.data.gouv.fr/api/1/datasets/lieux-de-vaccination-contre-la-covid-19/"
+)
+
+
+def job(dataset_url="", verbose=False, is_scheduler=False):
+    try:
+        errors, imported, skipped = do_import(dataset_url, is_scheduler)
+
+        if is_scheduler:
+            _send_report(errors, imported, skipped)
+        else:
+            if verbose and len(errors) > 0:
+                print("Erreurs rencontrées :")
```

Comment:
How about passing an optional `logger` kwarg, which would be a function in charge of either logging or printing to the console? This is a dependency we could pass around as an argument and call it directly without using `if` statements  🤔 

And if we don't want to bother passing this arg to each function calls needing it, we should maybe consider a class.